### PR TITLE
Query change

### PR DESCRIPTION
**Pull request by katbow**
update query to use a fragment, since convert is not usually used with mssql_ecto adapter. #1103

### CODE REVIEW

File: lib/healthlocker/slam/carer_connection.ex [hunk 1]
```diff
@@ -20,6 +20,7 @@ defmodule Healthlocker.Slam.CarerConnection do
   end
 
   defp validate_slam(changeset, options \\ []) do
+    IO.inspect changeset
```
Comment (SimonLab) [inline lib/healthlocker/slam/carer_connection.ex]: Not sure if you want to keep this log of the changeset

Comment (katbow) [reply lib/healthlocker/slam/carer_connection.ex]: Whoops, no I do not. I'll remove that.

**Comment by reddog** [comment]
Looks like Travis build failed due to database timeout.  See https://travis-ci.org/healthlocker/healthlocker/builds/282820604#L1593

We've seen this before and the fix was to re-run the test.  @SimonLab is it possible to re-run the test if it's required?

**Comment by katbow** [comment]
the build is rerunning now, so we'll see how it goes.

**Comment by katbow** [comment]
There's an update for travis status [here](https://www.traviscistatus.com/incidents/7tb9nm7j4zh7) as well.

**Comment by reddog** [comment]
@katbow Looks like Travis is struggling - thanks very much for persisting with it....!

File: lib/healthlocker/slam/carer_connection.ex [hunk 2]
```diff
@@ -40,13 +40,16 @@ defmodule Healthlocker.Slam.CarerConnection do
     query = from e in EPJSUser,
       where: e."Forename" == ^forename
         and e."Surname" == ^surname
-        and e."DOB" == ^date_of_birth
+        and e."DOB" == fragment("convert (datetime, ?, 103)", ^date_of_birth)
```
Comment (SimonLab) [inline lib/healthlocker/slam/carer_connection.ex]: image

:+1:

**Pull request merged**